skrifa/src/outline/native_hint.rs

```diff
@@ -0,0 +1,147 @@
+//! Support for applying embedded hinting instructions.
+
+use super::{
+    cff, AdjustedMetrics, DrawError, Hinting, LocationRef, NormalizedCoord, OutlineCollectionKind,
+    OutlineGlyph, OutlineGlyphCollection, OutlineKind, OutlinePen, Size,
+};
+
+/// Modes for native hinting.
+///
+/// Only the `glyf` format supports all hinting modes.
+#[derive(Copy, Clone, PartialEq, Eq, Debug)]
+pub enum NativeHinting {
```

Comment:
Do these mirror FreeType hinting modes? If so, we should link to it

Reply:
They do mostly but this type will be changing soon. @drott and I agreed to make the actual functionality of the hinting mode more explicit along with docs mapping to FT load flags. Coming with hinting PR.